web/frontend/package.json

```diff
@@ -23,15 +23,18 @@
     "google-protobuf": "^3.21.2",
     "three": "^0.145.0",
     "trzy": "^0.0.14",
-    "vue": "^3.2.41"
+    "vue": "^3.2.47"
   },
   "devDependencies": {
     "@mdi/js": "^6.9.96",
+    "@sveltejs/vite-plugin-svelte": "^2.0.3",
+    "@tsconfig/svelte": "^3.0.0",
     "@types/google-protobuf": "^3.15.6",
     "@types/google.maps": "^3.52.2",
     "@types/three": "^0.144.0",
     "@typescript-eslint/eslint-plugin": "^5.40.1",
-    "@vitejs/plugin-vue": "^3.1.2",
+    "@vitejs/plugin-vue": "^4.0.0",
```

Comment:
I bumped up vite-related & vue infrastructure while working on this. It ended up fixing a few typescript issues we were encountering.